README.md

```diff
@@ -18,6 +18,18 @@ disabled by specifying `--compression=none`.
 	cd docker2aci
 	./build.sh
 
+## Build with rkt
+
+assuming that you have rkt installed on you system
```

Comment:
Assuming that you have rkt installed on your system, this command will build docker2aci using it. The resulting binary will be in the `bin` folder .